app/Labyrinth/labyrinth.h

```diff
@@ -12,12 +12,31 @@
 #define LABYRINTH_H
 #include "../Common/common.h"
 #include "../Snake/snake.h"
+#include "../Change/change.h"
+#include "../Ball/ball.h"
 #include <ncurses.h>
 #include <string>
 #include <cstring>
 #include <cstdio>
 #include <list>
 
+/**
+ * @class DisplayHandler displayHandler.h
+ * @method displayFull
+ */
+class DisplayHandler
+{
+    public:
+        Labyrinth* labyrinth;
+        int prevDisplayMethod = 0;
+        void displayHandler(int displayMethod);
+        void displayFull();
+        void freeDisplay();
+        void forcedDisplay(int displayMethod);
+        void displayPartialy();
+        void displayUpdated();
+};
+
 
 /**
```

Comment:
Group into 1 comment with list

Reply:
how to do it without loosing doxygen integration?

Reply:
(:~{())